internal/service/streamconnection/resource_stream_connection_migration_test.go

```diff
@@ -24,12 +24,12 @@ func TestMigStreamRSStreamConnection_kafkaPlaintext(t *testing.T) {
 		Steps: []resource.TestStep{
 			{
 				ExternalProviders: mig.ExternalProviders(),
-				Config:            kafkaStreamConnectionConfig(projectID, instanceName, "user", "rawpassword", "localhost:9092,localhost:9092", "earliest", false),
+				Config:            kafkaStreamConnectionConfig(projectID, instanceName, "user", "rawpassword", "localhost:9092,localhost:9092", "earliest", false, false),
```

Comment:
Can we refactor these to use the `mig.CreateAndRunTest`?

Reply:
I don't think it makes sense to refactor each migration test to use `mig.CreateAndRunTest` because Config of step 1 and step 2 changes (1 step does not have `networking`, second step does), and from the code I see: `// CreateTest returns a new TestCase that reuses step 1 and adds a TestStepCheckEmptyPlan.`

Reply:
would question if we need to test the update as part of the migration test, checking an empty plan after adopting latest version is the regular case of migration tests.

Reply:
wanted to follow up here to confirm if an empty plan is encountered when upgrading to latest version, or do users need to add `networking` once they upgrade to obtain an empty plan? @EspenAlbert I see you included plan modifiers but not certain if it is intended for covering this case.

Reply:
My understanding is that those changes avoid plan changes if user does not add the `networking` attribute once they upgrade. But please @EspenAlbert confirm, also might be necessary to change the migration tests now that the plan does not change?

Reply:
Yes, it leads to no plan changes. Will push updates when we are unblocked